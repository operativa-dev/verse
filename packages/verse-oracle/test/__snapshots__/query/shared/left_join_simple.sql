-- Executing SQL: Parameters: [$1=3]
select t1.AlbumId, t1.Title, t1.ArtistId, t2.ArtistId c0, t2.Name
from Album t1 
left join Artist t2 on t1.ArtistId = t2.ArtistId
fetch next :0 rows only

