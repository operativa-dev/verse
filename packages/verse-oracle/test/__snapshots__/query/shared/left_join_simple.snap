[
  [
    Album {
      "albumId": 1,
      "artistId": 1,
      "title": "For Those About To Rock We Salute You",
    },
    Artist {
      "artistId": 1,
      "name": "AC/DC",
    },
  ],
  [
    Album {
      "albumId": 2,
      "artistId": 2,
      "title": "Balls to the Wall",
    },
    Artist {
      "artistId": 2,
      "name": "Accept",
    },
  ],
  [
    Album {
      "albumId": 3,
      "artistId": 2,
      "title": "Restless and Wild",
    },
    Artist {
      "artistId": 2,
      "name": "Accept",
    },
  ],
]